docs/reference-guides/filters/editor-filters.md

```diff
@@ -124,7 +128,30 @@ Default `true`. Indicates whether the user can access the code editor **in addit
 
 If set to false the user will not be able to switch between visual and code editor. The option in the settings menu will not be available and the keyboard shortcut for switching editor types will not fire.
 
-### Block Directory
+## Logging errors
+
+A JavaScript error in a part of the UI shouldn’t break the whole app. To solve this problem for users, React library uses a concept of an [“error boundary”](https://reactjs.org/docs/error-boundaries.html). Error boundaries are React components that catch JavaScript errors anywhere in their child component tree, and display a fallback UI instead of the component tree that crashed.
```

Comment:
The brief background about Error Boundaries in React really helps give some useful context before hoping into the hook! 👍🏻